src/Razor/test/Microsoft.NET.Sdk.Razor.Test/Microsoft.NET.Sdk.Razor.Test.csproj

```diff
@@ -55,14 +55,24 @@
     <ProjectReference Include="..\Microsoft.AspNetCore.Razor.Test.MvcShim.ClassLib\Microsoft.AspNetCore.Razor.Test.MvcShim.ClassLib.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.NET.Sdk.Razor\Microsoft.NET.Sdk.Razor.csproj" />
 
-    <!-- We don't need anything in this assembly, we just want to make sure it's built -->
+    <!-- We don't need anything in these assemblies, we just want to make sure they are built -->
     <ProjectReference Include="..\..\src\Microsoft.AspNetCore.Mvc.Razor.Extensions\Microsoft.AspNetCore.Mvc.Razor.Extensions.csproj" ReferenceOutputAssembly="false" />
   </ItemGroup>
 
   <PropertyGroup>
     <MSBuildLocationFileOutput>$(IntermediateOutputPath)BuildVariables.generated.cs</MSBuildLocationFileOutput>
+    <BuildDependsOn>_BuildPackageAssets</BuildDependsOn>
   </PropertyGroup>
 
+  <Target Name="_BuildPackageAssets" AfterTargets="Build">
+    <ItemGroup>
+      <_ProjectsToPack Include="..\testapps\PackageLibraryTransitiveDependency\PackageLibraryTransitiveDependency.csproj" />
+      <_ProjectsToPack Include="..\testapps\PackageLibraryDirectDependency\PackageLibraryDirectDependency.csproj" />
+    </ItemGroup>
+
+    <Msbuild Projects="@(_ProjectsToPack)" Targets="Pack" BuildInParallel="false" />
```

Comment:
note: this won't work with the VS up to date check. This will require you to touch a file in the test project to trigger an out of date notification.